client/components/domains/connect-domain-step/index.jsx

```diff
@@ -335,7 +335,7 @@ function ConnectDomainStep( {
 			return <div className={ baseClassName + '__sidebar-placeholder' }></div>;
 		}
 
-		if ( ! isStepStart ) {
+		if ( ! isStepStart || ! domainSetupInfo?.data?.is_supported_tld ) {
```

Comment:
This is to hide this box, on the right, to avoid suggesting an unsupported action

![image](https://github.com/user-attachments/assets/8ca46d1d-c18e-41f4-8218-a3d55cd90f7f)